pkg/agent/client.go

```diff
@@ -21,24 +21,32 @@ import (
 	"time"
 
 	"github.com/gammazero/workerpool"
+	"google.golang.org/protobuf/types/known/emptypb"
+
 	serverutils "github.com/livekit/livekit-server/pkg/utils"
 	"github.com/livekit/protocol/livekit"
 	"github.com/livekit/protocol/logger"
 	"github.com/livekit/protocol/rpc"
 	"github.com/livekit/protocol/utils"
 	"github.com/livekit/psrpc"
-	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 const (
 	EnabledCacheTTL         = 1 * time.Minute
 	RoomAgentTopic          = "room"
 	PublisherAgentTopic     = "publisher"
+	ParticipantAgentTopic   = "participant"
 	DefaultHandlerNamespace = ""
 
 	CheckEnabledTimeout = 5 * time.Second
 )
 
+var jobTypeTopics = map[livekit.JobType]string{
+	livekit.JobType_JT_ROOM:        RoomAgentTopic,
+	livekit.JobType_JT_PUBLISHER:   PublisherAgentTopic,
+	livekit.JobType_JT_PARTICIPANT: ParticipantAgentTopic,
```

Comment:
Is there a chance of `PUBLISHER` and `PARTICIPANT` colliding? i. e. can a `PUBLISHER` be assigned a `PARTiCIPANT` job also?

Reply:
yes, if there are both publisher and participant dispatches in the same room the publishers will get one of each